.github/workflows/deploy-docs.yml

```
@@ -45,8 +64,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Deploy docs (production)
-        if: github.ref == 'refs/heads/master'
-        run: yarn deploy:prod
+        if: github.ref == 'refs/heads/main'
```

```suggestion
        if: github.ref == 'refs/heads/master'
```

React Live is still using `master` branch.